import-service/handlers/importFileParser.ts

```diff
@@ -0,0 +1,67 @@
+import { S3Event } from 'aws-lambda';
+import { S3, SQS } from 'aws-sdk';
+import parse from 'csv-parser';
+
+const importFileParser = async (
+  event: S3Event
+): Promise<{ statusCode: number }> => {
+  try {
+    const s3 = new S3({ region: 'eu-west-1' });
+    const sqs = new SQS({ region: 'eu-west-1' });
+    const parser = parse({
+      separator: ';',
+    });
+
+    for (const record of event.Records) {
```

Comment:
this part of the file is hardly readable, consider to split it into smaller parts/functions to improve overall readability